.eslintignore

```diff
@@ -3,3 +3,4 @@ coverage*
 lib
 node_modules
 pnpm-lock.yaml
+pnpm-workspace.yaml
```

Comment:
Hmm, the template right now doesn't support monorepos/workspaces, and doesn't include any non-essential files in the `*ignore` files right now. I'm averse to adding in any neighboring / often-likely-to-used files. The list of potential ones is pretty big and it can be hard to precisely define what should or shouldn't be added. And anything that isn't relevant to a user's project is -potentially confusing- extra cruft for the project.

Maybe this should be an FAQ entry for now (sibling issue if so: #942)? Thoughts?

Reply:
It adds annoyance to add these ignores and doesn't hurt the setup


I don't like the FAQ-idea, most people will know what to do but it's just annoying to do